### PR TITLE
Mike1

### PR DESCRIPTION
**Pull request by MikeLeeJP**

### CODE REVIEW

File: yourSolution.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
 // Name
```
Comment (dsope05) [inline yourSolution.js]: please include name and problem # and description here

Comment (dsope05) [inline yourSolution.js]: also change title of file to "problem#".js aka 13.js, if it is taken, try 13a.js for example

File: solutions/test.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+//this is a test file
```
Comment (dsope05) [inline solutions/test.js]: remove this empty file from your commit, does this algorithm already have a test written for it?

Comment (dsope05) [inline solutions/test.js]: please include issue # in the title instead of Mike1

**Comment by yjlim5** [comment]
closing due to inactivity

**Pull request closed without merging**